Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -38,8 +39,13 @@ using tstring = std::string;
 
 namespace vtStor
 {
-	using DeviceHandle = int;
-	using HANDLE = int;
+
+struct DeviceHandle
```

Comment:
This approach isn't good. It's going to introduce an inconsistent issue. Windows and Linux get different structure of DeviceHandle.